composables/useErc725.ts

```diff
@@ -1,33 +1,38 @@
 import { ERC725, ERC725JSONSchema } from '@erc725/erc725.js'
 import LSP3UniversalProfileMetadata from '@erc725/erc725.js/schemas/LSP3UniversalProfileMetadata.json'
+import LSP4DigitalAsset from '@erc725/erc725.js/schemas/LSP4DigitalAsset.json'
 import { LSP3Profile } from '@lukso/lsp-factory.js'
 import Web3 from 'web3'
+import { LSP4DigitalAssetJSON } from '@lukso/lsp-factory.js/build/main/src/lib/interfaces/lsp4-digital-asset'
```

Comment:
Im not sure we want to have a "link" here to lsp-factory.js 

Im not sure if it is lsp-factory.js 's responsibility to provide these interfaces.

It is very handy, but maybe these interfaces could have a better "home" than lsp-factory.js

@CallumGrindle what do you think?

Reply:
yeah these typings are handy but it seems overkill to have to import the whole lsp-factory just for metadata types. I think we can move the types + the LSP6 methods from erc725.js into some kind of lsp utils lib. There isnt loads but there starts to be enough for its own lib

Reply:
idd we use package just for types here. I guess we should have npm package with just typings maybe